docs/modules/sql/pages/querying-streams.adoc

```diff
@@ -97,91 +116,190 @@ If the client stops receiving results, but doesn't close the result, Hazelcast w
 
 You can manage jobs backing SQL queries. For example, you may want to show all running jobs or cancel a query. See xref:pipelines:job-management.adoc[].
 
-[[aggregation]]
-== Stream Aggregation
+[[windowing]]
+== Windowing
```

Comment:
```suggestion
== Windowed Aggregations
```

Windowing means something different in SQL: https://learnsql.com/blog/sql-window-functions-examples/

Reply:
Updated as requested.